panopticum/fixtures/init.json

```diff
@@ -169,6 +169,22 @@
       "description": ""
     }
   },
+  {
+    "model": "panopticum.requirement",
+    "pk": 16,
+    "fields": {
+      "title": "API guideline compliance",
+      "description": ""
+    }
+  },
+  {
+    "model": "panopticum.requirement",
+    "pk": 17,
+    "fields": {
+      "title": "FIPS compliance",
+      "description": ""
+    }
+  },
```

Comment:
GDPR compliance is missing, it's one of the most important one

Reply:
It's public compliance?